src/EditorFeatures/CSharpTest/AddRequiredParentheses/AddRequiredParenthesesTests.cs

```diff
@@ -197,36 +205,16 @@ public async Task TestLogicalPrecedenceMultipleEqualPrecedenceParts1()
             await TestAsync(
 @"class C
 {
-    void M()
-    {
-        int x = a || b $$&& c && d;
-    }
-}",
-@"class C
-{
-    void M()
-    {
-        int x = a || (b && c && d);
-    }
-}", RequireAllParenthesesForClarity);
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddRequiredParentheses)]
-        public async Task TestLogicalPrecedenceMultipleEqualPrecedenceParts2()
-        {
-            await TestAsync(
-@"class C
-{
-    void M()
+    void M(bool a, bool b, bool c, bool d)
     {
-        int x = a || b && c $$&& d;
+        bool x = a || b [|&&|] c [|&&|] d;
```

Comment:
📝 It's no longer possible to execute a _single_ code fix for a _specific_ diagnostic when more than one fixable diagnostic is reported. The test framework will iteratively fix each diagnostic in lexicographic order until one of the following occurs:

1. No diagnostics remain
2. No applicable code fixes are registered
3. The code fix fails to make a change